tutorial/README.md

```diff
@@ -0,0 +1,413 @@
+
+Listing 4 will register the `event` table as `myevent` table in the Spark environment.
+
+```scala
+val eventQuery = "select * from event"
+val eventDF = sqlContext.read
+             .format("com.databricks.spark.redshift")
+             .option("url",jdbcURL ) 
+             .option("tempdir", tempS3Dir)
+             .option("query", eventQuery) 
+             .load()   
+eventsDF.registerTempTable("myevent") 
+```
+Listing 4: Create a table myevent in the Spark environment
+
+Listing 5 demonstrates how to write the contents of `myevent` to a Redshift table `redshiftevent`
+
+```scala
+/*
+ *Create a new table redshiftevent after dropping any existing redshiftevent table
+ *and write event records with event id less than 1000
+ */
+sqlContext.sql("select * from myevent where eventid<=1000").withColumnRenamed("eventid", "id")
+             .write.format("com.databricks.spark.redshift")
+            .option("url", jdbcURL)
+            .option("tempdir", tempS3Dir)
+            .option("dbtable", "redshiftevent")
+            .mode(SaveMode.Overwrite) 
+            .save()
+/*
+ *Append to an existing table redshiftevent if it exists or create a new one if it does not
+ *exist and write event records with event id greater than 1000
+ */
+
+sqlContext.sql("select * from myevent where eventid>1000").withColumnRenamed("eventid", "id")
+          .write.format("com.databricks.spark.redshift")
+          .option("url", jdbcURL) 
+          .option("tempdir", tempS3Dir) 
+          .option("dbtable", "redshiftevent") 
+          .mode(SaveMode.Append)
+          .save()
+```
+Listing 5: Writing to Redshift tables
+
+spark-redshift automatically creates a Redshift table with the appropriate schema determined from the table/DataFrame being written. The default behavior is to create a new table and to throw an error message if a table with the same name already exists (corresponding to `SaveMode.ErrorIfExists`)
+
+There are two key points to note in Listing 5-
+
+1. Note the `.withColumnRenamed("eventid", "id")`. This feature if necessary if any of the source table columns names are key words in Redshift (ex. table) then spark-redshift will throw and error. This will be [fixed](https://github.com/databricks/spark-redshift/issues/80) in the next release 
+2. Note how we utilize the modes. The first write utilizes the mode `SaveMode.Overwrite` which means that the table will be dropped and recreated it exists. The second query utilizes `SaveMode.Append` which will create the table if it does not exist but will append to it if it does. The default mode is `SaveMode.ErrorIfExists` which creates the table if it does not exists and throws and error if it does. The last mode is `SaveMode.Ignore` which is same as `SaveMode.Overwrite` if the table does not exist but does nothing if the table exists.
```

Comment:
I hate the word "utilize" and wish it would be banned from all technical writing. It tends to give writing an awkward formality and is almost never better than "use" in my opinion.